discovery-infra/utils.py

```diff
@@ -126,7 +126,7 @@ def is_cvo_available():
 
 
 def get_libvirt_nodes_macs(network_name):
-    return get_libvirt_nodes_mac_role_ip_and_name(network_name).keys()
```

Comment:
lets just move usage to test_infra/utils? for those functions? and delete them from here? Will help a lot

Reply:
It's already there, as I mentioned. Just change the imports and it will continue to work. There are other functions in use from the wrong utils, so I couldn't change the imports in this PR without making it even larger.

https://github.com/openshift/assisted-test-infra/pull/290/files#diff-61d18d81e7d4ae0105d0475fc2abd8dc0a66b687df5e35c56c8a337fbd7a7e4eL108